src/combineActions.js

```diff
@@ -0,0 +1,28 @@
+import isString from 'lodash/isString';
+import isFunction from 'lodash/isFunction';
+import isEmpty from 'lodash/isEmpty';
+import toString from 'lodash/toString';
+import isSymbol from 'lodash/isSymbol';
+
+export const ACTION_TYPE_DELIMITER = '|redux-action-delimiter|';
```

Comment:
It's used as a special character to join actions, then [split them again in the reducer](https://github.com/acdlite/redux-actions/pull/113/files#diff-b35552629f82c2c3dd19251a9cec2615R8).

This technique was [proposed by the reporter in the original issue](https://github.com/acdlite/redux-actions/issues/31#issue-107937867).